packages/web3-errors/src/web3_error_base.ts

```diff
@@ -18,16 +18,50 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 /* eslint-disable max-classes-per-file */
 
 import { Web3Error } from 'web3-types';
+import { ERR_MULTIPLE_ERRORS } from './error_codes.js';
 
+/**
+ * Base class for Web3 errors.
+ */
 export abstract class BaseWeb3Error extends Error implements Web3Error {
 	public readonly name: string;
 	public abstract readonly code: number;
 	public stack: string | undefined;
-	public innerError: Error | Error[] | undefined;
```

Comment:
This will break if any one using `innerError`, why not retain it and remove in next major version of errors? 
so for now `innerError` will be copy of `cause`.